README.md

```diff
@@ -55,8 +55,13 @@ These contributions are **NOT** supported by the **nanoFramework** team and any
   <td><a href="libraries/LoRa-RFM9X">Library for controlling a LoRa HopeRF RFM9X</a></td>
   <td><a href="libraries/LoRa-RFM9X">Library for controlling a LoRa RAK811</a></td> 
   <td><a href="libraries/ThingsBoardDotNet">Library to work with ThingsBoard IoT Platform</a></td> 
-  <td><a href="libraries/nanoFramework.CoAP">nanoFramework.CoAP library</a></td>
-  <td><!--<a href="Utility/util3">Utility Three</a>--></td>
+  <td><a href="libraries/nanoFramework.CoAP">nanoFramework.CoAP library</a></td>  
+ </tr>
```

Comment:
I know you'll call me picky but can you please also properly align the spaces for the tables.
Thanks a lot.

Reply:
Tabs vs Spaces :-) I just copy and pasted additional rows and didn't pay attention to formatting. :-) Fixed.